netbox/ipam/tables/ip.py

```diff
@@ -360,8 +368,8 @@ class IPAddressTable(TenancyColumnsMixin, NetBoxTable):
         orderable=False,
         verbose_name='NAT (Inside)'
     )
-    nat_outside = tables.Column(
-        linkify=True,
+    nat_outside = tables.TemplateColumn(
```

Comment:
We can use [ManyToManyColumn](https://github.com/netbox-community/netbox/blob/646272f9b39aec3edf58f1c59c9f72821ba099ac/netbox/netbox/tables/columns.py#L79) here instead of creating a custom template for the column.

Reply:
The ManyToManyColumn doesn't seem to function correctly here.
```
nat_outside = tables.ManyToManyColumn(
        linkify=True,
        orderable=False,
        verbose_name='NAT (Outside)'
    )
```
This produces the correct value in the table, but the links are not correct. They link to the IP address instead of the NAT address clicked.
![image](https://user-images.githubusercontent.com/52260120/186014696-2a0626e7-5114-4c7b-a2d4-3d3b63d06d6a.png)

Reply:
Ah, got it -> linkify_item=True,